### ISSUE TITLE
make keyPrefix default to $app->id for host-shared caches like Apc, Memcache

### ISSUE THREAD

**Issue by dicrtarasov** [yiisoft/yii-demo#22]
For shared caches, like APCu, Memcache, the defaut behaviour require to use keyPrefix, because of interpolability of hosting application, which share caches. So, defult configuration is not usabled.

I'm proposing set keyPrefix to \Yii::$app->id by default, so, the default configuration of \yii\cache\ApcCache, \yii\cache\MemCache become usable.

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.20
| PHP version      | 7.2
| Operating system | debian linux

**Comment by samdark** [comment]
Could be done in the default app config.

**Comment by dicrtarasov** [comment]
yes! for ex. as Requres::cookieValidationKey is generated in default config when installing :)
The difference is, the Cache::prefixKey is better in human-readable variant.
